README.md

```diff
@@ -12,7 +12,33 @@ The BaseView framework provides an organizational tool for writing custom views
 
 ## Installation
 
-- Add the `BaseView.swift` file to your Xcode project.
+### CocoaPods
+
+[CocoaPods](http://cocoapods.org) is a dependency manager for Cocoa projects. You can install it with the following command:
+
+```bash
+$ gem install cocoapods
+```
+
+To integrate BaseViewSwift into your Xcode project using CocoaPods, specify it in your `Podfile`:
```

Comment:
Correct, I am referring to it as the framework (i.e. package) rather than the file. Even though it is just a one file framework right now, it may not be in the future. It also aligns with verbage from Alamofire and other repos I used as an example.